frontend/src/scenes/data-management/DataManagementScene.tsx

```diff
@@ -0,0 +1,211 @@
+> = {
+    [DataManagementTab.EventDefinitions]: {
+        url: urls.eventDefinitions(),
+        label: 'Events',
+        buttons: <NewActionButton />,
+        content: <EventDefinitionsTable />,
+    },
+    [DataManagementTab.Actions]: {
+        url: urls.actions(),
+        label: (
+            <TitleWithIcon
+                icon={
+                    <Tooltip title="Actions consist of one or more events that you have decided to put into a deliberately-labeled bucket. They're used in insights and dashboards.">
+                        <IconInfo />
+                    </Tooltip>
+                }
+                data-attr="data-management-actions-tab"
+            >
+                Actions
+            </TitleWithIcon>
+        ),
+        content: <ActionsTable />,
+    },
+    [DataManagementTab.PropertyDefinitions]: {
+        url: urls.propertyDefinitions(),
+        label: (
+            <TitleWithIcon
+                icon={
+                    <Tooltip title="Properties are additional data sent along with an event capture. Use properties to understand additional information about events and the actors that generate them.">
+                        <IconInfo />
+                    </Tooltip>
+                }
+                data-attr="data-management-event-properties-tab"
+            >
+                Properties
+            </TitleWithIcon>
+        ),
+        content: <PropertyDefinitionsTable />,
+    },
+    [DataManagementTab.Annotations]: {
+        url: urls.annotations(),
+        content: <Annotations />,
+        label: 'Annotations',
+        buttons: <NewAnnotationButton />,
+    },
+    [DataManagementTab.History]: {
+        url: urls.dataManagementHistory(),
+        label: 'History',
+        content: (
+            <ActivityLog
+                scope={ActivityScope.DATA_MANAGEMENT}
+                caption={
+                    'Only actions taken in the UI are captured in History. Automatic creation of definitions by ingestion is not shown here.'
+                }
+            />
+        ),
+    },
+    // TODO: Only show this if it should be shown
+    [DataManagementTab.IngestionWarnings]: {
+        url: urls.ingestionWarnings(),
+        label: 'Ingestion Warnings',
+        content: <IngestionWarningsView />,
+    },
+    [DataManagementTab.Database]: {
+        url: urls.database(),
+        label: (
+            <>
+                Database
+                <LemonTag type="warning" className="uppercase ml-2">
+                    Beta
+                </LemonTag>
+            </>
+        ),
+        content: <DatabaseTableList />,
+    },
+}
+
+const dataManagementSceneLogic = kea<dataManagementSceneLogicType>({
+    path: ['scenes', 'events', 'dataManagementSceneLogic'],
+    connect: {
+        values: [featureFlagLogic, ['featureFlags']],
+    },
+    actions: {
+        setTab: (tab: DataManagementTab) => ({ tab }),
+    },
+    reducers: {
+        tab: [
```

Comment:
This is so much snappier than each tab being its own scene 👏

Reply:
Actually got some ideas for 3000-ifying this tab-scene type world as I think it is pretty abstractable.